Sources/tasks/ApprovePost-Notify.php

```diff
@@ -91,8 +91,8 @@ public function execute()
 					'id_member' => $member,
 					'id_member_started' => $posterOptions['id'],
 					'member_name' => $posterOptions['name'],
-					'content_type' => 'topic',
-					'content_id' => $topicOptions['id'],
+					'content_type' => $type == 'topic' ? 'topic' : 'msg',
```

Comment:
Seems to me like this ternary is not needed.

Reply:
Here, it might be 'topic' for new topics, or 'reply' for replies to topics.  For new topics, you need to be able to look up the topic alerts later.  For replies, you will need to look up message-specific alerts later.